_integrations/vim.md

```diff
@@ -6,8 +6,9 @@ github: omnisharp-vim
 ---
 
 1. Install the [OmniSharp-vim](https://github.com/OmniSharp/Omnisharp-vim) Vim/neovim plugin using your favourite method or plugin manager.
-1. Optionally install a linting plugin ([ALE](https://github.com/w0rp/ale) or [Syntastic](https://github.com/scrooloose/syntastic)) and selector plugin ([fzf.vim](https://github.com/junegunn/fzf.vim), [CtrlP](https://github.com/kien/ctrlp.vim) or [unite.vim](https://github.com/Shougo/unite.vim)).
-1. Create some [mappings](https://github.com/OmniSharp/Omnisharp-vim#configuration), or install [vim-sharpenup](https://github.com/nickspoons/vim-sharpenup) to get up and running quickly.
-1. Open a .cs file in vim. OmniSharp-vim will ask permission to install the server - answer 'Y', wait for it to do its thing, and you're away!
+1. Optionally install a linting plugin: [ALE](https://github.com/w0rp/ale)
+3. Optionally install selector plugin: [fzf.vim](https://github.com/junegunn/fzf.vim) or [CtrlP](https://github.com/kien/ctrlp.vim)
+5. Create some [mappings](https://github.com/OmniSharp/Omnisharp-vim#configuration), or install [vim-sharpenup](https://github.com/nickspoons/vim-sharpenup) to get up and running quickly.
+6. Open a .cs file in vim. OmniSharp-vim will ask permission to install the server - answer 'Y', wait for it to do its thing, and you're away!
```

Comment:
```suggestion
1. Optionally install selector plugin: [fzf.vim](https://github.com/junegunn/fzf.vim) or [CtrlP](https://github.com/kien/ctrlp.vim)
1. Create some [mappings](https://github.com/OmniSharp/Omnisharp-vim#configuration), or install [vim-sharpenup](https://github.com/nickspoons/vim-sharpenup) to get up and running quickly.
1. Open a .cs file in vim. OmniSharp-vim will ask permission to install the server - answer 'Y', wait for it to do its thing, and you're away!
```
I understand numbering does not matter in MD numbered list, but IMO either keep all at one or properly order for readability, thx!